.github/workflows/build.yml

```diff
@@ -254,6 +254,26 @@ jobs:
             docker.io/hashicorp/${{env.repo}}:${{env.version}}
             public.ecr.aws/hashicorp/${{env.repo}}:${{env.version}}
 
+  enos-test-docker:
+    name: Enos Docker
+    # Only run the Enos workflow against branches that are created from the
+    # hashicorp/vault repository. This has the effect of limiting execution of
+    # Enos scenarios to branches that originate from authors that have write
+    # access to hashicorp/vault repository. This is required as Github Actions
+    # will not populate the required secrets for branches created by outside
+    # contributors in order to protect the secrets integrity.
+    if: "! github.event.pull_request.head.repo.fork"
```

Comment:
It's a bummer that we have this constraint for a scenario that doesn't require AWS credentials but does require TFC credentials.

Reply:
Yeah, I guess if we ever open source enos-provider this might not be required.